src/test/java/org/folio/service/dataimport/handlers/CreateOrderEventHandlerTest.java

```diff
@@ -1389,6 +1390,44 @@ public void shouldCreatePendingOrderAndNotMapVendorMaterialSupplierAndAccessProv
     assertNull(poLine.getEresource());
   }
 
+  @Test
+  public void shouldNotReturnFailedByDuplicateEventExceptionFutureWhenRecordIdIsDifferent(TestContext context) {
```

Comment:
propose to update this test in a way so it checks events deduplication behavior, it will also increase tests coverage

Reply:
done